Task1.1/task1.js

```diff
@@ -0,0 +1,16 @@
+const through2 = require("through2");
+
+const reverseString = () => {
+  console.log("Please enter any string to reverse.\n");
+  process.stdin
+    .pipe(
+      through2(function write(buffer, encoding, next) {
```

Comment:
Can you do this without the use of external modules?

Reply:
Fixed